opts.go

```diff
@@ -47,6 +47,7 @@ var gOpts struct {
 	drawbox            bool
 	dupfilefmt         string
 	globsearch         bool
+	globfilter         bool
```

Comment:
Place this above `globsearch` to maintain alphabetical order. Same for the below definition.